tests/algorithms/test_mce_irl.py

```diff
@@ -314,6 +314,31 @@ def test_tabular_policy(rng):
     np.testing.assert_equal(timesteps[0], 2 - mask.astype(int))
 
 
+def test_tabular_policy_rollouts(rng):
+    """Tests tabular policy prediction, especially timestep calculation and masking."""
+    state_space = gym.spaces.Discrete(5)
+    action_space = gym.spaces.Discrete(3)
+
+    s_a = np.stack([np.eye(3)] * 5, axis=1)
+    pi = np.concatenate([s_a] * 7, axis=0)
```

Comment:
I think this does the same and seems more canonical? (I don't particularly care either way though.)
```suggestion
    pi = np.repeat(s_a, 7, axis=0)
```